---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Metabolic rules of microbial community assembly
subtitle: ''
summary: ''
authors:
- Sylvie Estrela
- Jean C C Vila
- Nanxi Lu
- Djordje Bajic
- Maria Rebolleda-Gomez
- Chang-Yu Chang
- Alvaro Sanchez
tags: []
categories: []
date: '2020-03-01'
lastmod: 2021-08-25T14:50:46-04:00
featured: false
draft: false

links:
- name: Link
  url: https://doi.org/10.1101/2020.03.09.984278
  icon_pack: fas
  icon: link
- name: Preprint
  url: https://www.biorxiv.org/content/10.1101/2020.03.09.984278v1
  icon_pack: fa
  icon: file

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-08-25T18:50:46.584472Z'
publication_types:
- '3'
abstract: To develop a quantitative theory that can predict how microbiomes assemble,
  and how they respond to perturbations, we must identify which descriptive features
  of microbial communities are reproducible and predictable, which are unpredictable,
  and why. The emergent metagenomic structure of communities is often quantitatively
  convergent in similar habitats, with highly similar fractions of the metagenome
  being devoted to the same metabolic pathways. By contrast, the species-level taxonomic
  composition is often highly variable even in replicate environments. The mechanisms
  behind these patterns are not yet understood. By studying the self-assembly of hundreds
  of communities in replicate, synthetic habitats, we show that the reproducibility
  of microbial community assembly reflects an emergent metabolic structure, which
  is quantitatively predictable from first-principles, genome-scale metabolic models.
  Taxonomic variability within functional groups arises through multistability in
  population dynamics, and the species-level community composition is predictably
  governed by the mutual competitive exclusion of two sub-dominant strains. Our findings
  provide a mechanistic bridge between microbial community structure at different
  levels of organization, and show that the evolutionary conservation of metabolic
  traits, both in terms of growth responses and niches constructed, can be leveraged
  to quantitatively predict the taxonomic and metabolic structure of microbial communities.
publication: '*bioRxiv*'
---
